### PR TITLE
Update identify-a-hold-on-an-exchange-online-mailbox.md

### PR DESCRIPTION
**Pull request by guruxp**
Including the Hold type values as part of the maibox diagnostic logs output as they were missing.

### CODE REVIEW

File: microsoft-365/compliance/identify-a-hold-on-an-exchange-online-mailbox.md [hunk 1]
```diff
@@ -281,6 +281,7 @@ Use the following table to help you understand each of the previous values liste
 |:------- |:----------- |
 | **ed**  | Indicates the End date, which is the date the retention policy was disabled. MinValue means the policy is still assigned to the mailbox. |
 | **hid** | Indicates the GUID for the retention policy. This value will correlate to the GUIDs that you collected for the explicit or organization-wide retention policies assigned to the mailbox.|
+| **ht**  | Indicates the hold type. '0' for LitigationHold; '1' for InPlaceHold; '2' for ComplianceTagHold; '3' for DelayReleaseHold; '4' for OrganizationRetention; '5' for CompliancePolicy; '6' for SubstrateAppPolicy and '7' for SharepointPolicy |
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/identify-a-hold-on-an-exchange-online-mailbox.md]:
```suggestion
| **ht**  | Indicates the hold type; **0** for LitigationHold, **1** for InPlaceHold, **2** for ComplianceTagHold, **3** for DelayReleaseHold, **4** for OrganizationRetention, **5** for CompliancePolicy, **6** for SubstrateAppPolicy, and **7** for SharepointPolicy. |
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @guruxp, kindly commit the suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by guruxp** [comment]
@JohanFreelancer9 the branch was outdated, could you please merge again?

**Comment by JohanFreelancer9** [comment]
> @JohanFreelancer9 the branch was outdated, could you please merge again?

@guruxp, I only did a copy edit. The changes still need to be reviewed, approved, and then merged by the document owner or manager. Unfortunately, I cannot merge them.

**Comment by yogkumgit** [comment]
@robmazz Please merge this PR. Thanks

**Comment by guruxp** [comment]
@yogkumgit any update on this please?

**Comment by guruxp** [comment]
@robmazz is there anything missing for this merge?

**Comment by robmazz** [comment]
@guruxp - thanks for the feedback. I'll confirm with the PG team and add if applicable. Thanks again.

**Pull request closed without merging**